### ISSUE TITLE
add NON-NLS fix wrong line

### ISSUE THREAD

**Issue by fnicastri** [hpoul/string_literal_finder#3]
When select `Add // NON-NLS` fix in Android Studio
often the comment is added to the wrong line
![image](https://user-images.githubusercontent.com/5978520/215129729-27463d7d-1a20-4ec9-ae30-7cb5dcb9a1b8.png)

**Comment by hpoul** [comment]
Yes, unfortunately the insertion fix is pretty dumb right now.. It just looks for a semicolon in the current line, and if it can't find one it looks for the previous one..

**Comment by fnicastri** [comment]
what about change it like the other `// ignore: this` 
the analyzer use just at the previous line?

**Comment by fnicastri** [comment]
@hpoul 
I would like to try to modify it but I need some directions on how to use the local copy,
possibly on one of my projects.

**Comment by fnicastri** [comment]
@hpoul are you interested?
